### PR TITLE
feat: streamline local Docker setup

### PR DESCRIPTION
**Pull request by w3b6x9**
## What kind of change does this PR introduce?

chore

## What is the new behavior?

Local Docker setup for multi-tenant Realtime is a breeze.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ seed:
 	mix run priv/repo/seeds.exs
 
 prod:
-	MIX_ENV=prod FLY_APP_NAME=realtime-local API_KEY=dev SECURE_CHANNELS=true API_JWT_SECRET=dev METRICS_JWT_SECRET=dev FLY_REGION=fra FLY_ALLOC_ID=123e4567-e89b-12d3-a456-426614174000 DB_ENC_KEY="1234567890123456" SECRET_KEY_BASE=M+55t7f6L9VWyhH03R5N7cIhrdRlZaMDfTE6Udz0eZS7gCbnoLQ8PImxwhEyao6D DASHBOARD_USER=realtime_local DASHBOARD_PASSWORD=password ERL_AFLAGS="-kernel shell_history enabled" iex -S mix phx.server
```
Comment (w3b6x9) [inline Makefile:11]: **REMINDER:** `API_KEY` is no longer used so remove it from staging and prod.

File: Dockerfile [hunk 2]
```diff
@@ -112,7 +112,7 @@ RUN mkdir -p /var/run/tailscale /var/cache/tailscale /var/lib/tailscale
 COPY limits.sh /app/limits.sh
 ENTRYPOINT ["/app/limits.sh"]
 
-CMD ["/tailscale/wrapper.sh"]
+CMD ["sh", "-c", "/tailscale/wrapper.sh && /app/bin/migrate && /app/bin/server"]
```
Comment (abc3) [inline Dockerfile:115]: We should check the Fly's behavior in the case when migration unsuccessful

File: docker-compose.dbs.yml [hunk 3]
```diff
@@ -1,16 +1,14 @@
 version: '3'
+
 services:
   db:
-    image: supabase/postgres:14.1.0
+    image: supabase/postgres:14.1.0.105
```
Comment (abc3) [inline docker-compose.dbs.yml:5]: Is there postgres v15 already available?

File: tailscale/wrapper.sh [hunk 4]
```diff
@@ -9,6 +9,3 @@ if [ "${ENABLE_TAILSCALE}" = true ]; then
     /tailscale/tailscaled --state=/var/lib/tailscale/tailscaled.state --socket=/var/run/tailscale/tailscaled.sock &
     /tailscale/tailscale up --authkey=${TAILSCALE_AUTHKEY} --hostname="${TAILSCALE_APP_NAME}" --accept-routes=true
 fi
-
-echo "Starting Realtime"
-sudo -E -u nobody /app/bin/server
```
Comment (abc3) [inline tailscale/wrapper.sh:14]: Let's rename this file if it is not a wrapper anymore

Comment (w3b6x9) [reply tailscale/wrapper.sh:14]: @abc3 it's still a wrapper for tailscale though and it's name-spaced appropriately: `tailscale/wrapper.sh`

File: docker-compose.dbs.yml (hunk 3 above)
Comment (w3b6x9) [reply docker-compose.dbs.yml:5]: @abc3 Yes, but most projects are on pg v14 so we should stick with it for the time being.

File: Dockerfile (hunk 2 above)
Comment (w3b6x9) [reply Dockerfile:115]: @abc3 looks like Fly's release command "perform database migrations" and "run in a temporary VM that is created just before your application is deployed and released" so it helps with "zero-downtime deploys" but what about local development or self-hosting that's not on Fly? How will we run db migrations then?

Comment (w3b6x9) [reply Dockerfile]: @abc3 I added back the release command and will take care of migrations `docker-compose.yml`.

**Review by abc3 — APPROVED** [review]

**Pull request merged**